### DISCUSSION TITLE
Improving the accessibility of documentation for new users

### DISCUSSION THREAD

**Discussion by fredjaya** [cogent3/cogent3#1677] [Ideas]
I think the cogent3 docs will be more accessible for new users with a "Getting Started" or "Quick Start" section.

I find it unclear where to go in the docs unless I already know what to look for, and a Getting Started section might be useful in providing a hands-on and (very) high-level overview of cogent3 and its features. 

Some common things across the examples below are:
- Fastest way to install and use the program (i.e. Jupyter in numpy) with a simple example
- Basic introduction into key features of the program (i.e. arrays in numpy, could be apps for Cogent3)
-  Links to more complex/in-depth docs throughout

Something else to consider is restructuring the documentation in order of complexity, e.g.:
1. Getting started/quick start
2. Installation 
3. Intro to cogent3 (most important data structures and functions)
4. Tutorials
5. Most other things...
6. API reference
7. Developer docs

**Examples**
- [NumPy: the absolute basics for beginners](https://numpy.org/doc/stable/user/absolute_beginners.html)
- [pandas: Getting Started](https://pandas.pydata.org/getting_started.html)
- [biopython: Getting Start](https://biopython.org/wiki/Getting_Started)

**Comment by GavinHuttley** [comment]
These are great suggestions @fredjaya ! Will follow up next year.

**Comment by KatherineCaley** [comment]
Another example is numba’s [5 minute guide](https://numba.readthedocs.io/en/stable/user/5minguide.html)
